docs/admin/troubleshooting_network.rst

```diff
@@ -2,3 +2,218 @@ Troubleshooting network issues
+can access SecureDrop from Tails.
+
+If you are certain that your credentials are correct but you are unable to log
+in, proceed to the next step.
+
+Step 3: Verify that all required VMs are running
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The following VMs must be running for all actions requiring network connectivity
+to work (e.g., logging in, checking for messages, downloading documents, replying
+to sources, starring sources, deleting sources):
+
+- ``sd-app``
+- ``sd-gpg``
+- ``sd-log``
+- ``sd-proxy``
+- ``sd-whonix``
+- ``sys-firewall``
+- ``sys-net``
+- ``sys-whonix`` (during updates)
+
+You can verify whether a VM is running or not by clicking the "Q" icon in the
+system tray (top right). Only VMs that are currently running will appear in the
+list:
+
+**[SCREENSHOT: Q widget with VM list]**
+
+If a required VM is not running, you can launch it from the Qube Manager. Open
+the Qube Manager by clicking **Open Qube Manager** in the menu above. A window
+like the following should appear:
+
+**[SCREENSHOT: Qube manager screenshot]**
+
+To start a VM, select it from the list, right-click it, and click **Start/Resume
+Qube**. Alternatively, you can click the "Play" button in the toolbar.
+
+In ordinary use, VMs required by SecureDrop should be started on boot or when
+they are needed. If you repeatedly experience problems with a necessary VM not
+running, or if an error message is displayed when attempting to start the VM,
+please contact us for assistance.
+
+If all required VMs are running, proceed to the next step.
+
+Step 4: Verify that required VMs have connectivity
+--------------------------------------------------
+In step 1, you have already verified that you can connect to the
+Internet using ``sys-net``. Now, test whether ``sys-firewall``, ``sd-whonix``
+and ``sd-proxy`` are working.
+
+First, open a terminal in ``sys-firewall`` and run the ``ping google.com`` command.
+You should see similar as in ``sys-net`` before.
+
+Now, open a terminal in ``sd-whonix`` and run the following command:
+
+``curl -s https://check.torproject.org/ | cat | grep -m 1 "Congratulations"``
+
+This command contacts a service intended for web browsers to verify whether your
+Tor connection is working.
+
+You should see the text "Congratulations. This browser is configured to use Tor."
+or a similar message on the terminal.
+
+If this command fails, proceed to the next step.
+
+If the command succeeds in ``sd-whonix``, run exactly the same command in
+``sd-proxy``. If it only fails in ``sd-proxy``, your workstation may be
+misconfigured, or the proxy may have crashed. In that case, skip ahead to step 6.
+We also recommend that you contact us, so we can help identify the root cause.
+
+Step 5: Restart Tor
+-------------------
+If you have narrowed down the problem to ``sd-whonix``, try restarting Tor.
+
+To do so, right-click the Tor icons in the top right corner of your Qubes
+desktop. They look like this:
```

Comment:
It'd be a shame not to reference the GUI icons, but given that we were using the terminal in `sd-whonix` in Step 4, it might simpler for an Admin to run `sudo systemctl restart tor`. We can take a closer look once the screenshots are proposed, but on my system, the `sd-whonix` / `sys-whonix` combination makes for a very confusing tooltray experience. If I were following the instructions, I might end up bouncing tor in `sys-whonix` rather than `sd-whonix` without realizing it.

Reply:
You're right, it's a massive simplification in this context to just do it on the terminal. If Tor issues are frequent it'd be great to be able to teach journalists how to use the widget, but given the current UX issues with it, we may just want to steer clear for now, also pending a final decision whether we'll continue to use Whonix.

Cross-ref: 
- icon glitchiness: https://github.com/QubesOS/qubes-issues/issues/5704 - there's a PR for it so hopefully will be resolved soon
- widget redesign proposal: https://phabricator.whonix.org/T963